tap_twilio/streams.py

```diff
@@ -12,313 +12,125 @@
-                        'api_version': '2010-04-01',
-                        'path': 'Accounts/{AccountSid}/AvailablePhoneNumbers/{ParentId}/Mobile.json',
-                        'data_key': 'available_phone_numbers',
-                        'key_properties': ['iso_country', 'phone_number'],
-                        'replication_method': 'FULL_TABLE',  # ALL for parent Address
-                        'params': {},
-                        'pagingation': 'root',
-                        'activate_version': True
-                    },
-                    # pylint: disable=line-too-long
-                    # Reference: https://www.twilio.com/docs/phone-numbers/api/availablephonenumber-tollfree-resource#read-multiple-availablephonenumbertollfree-resources
-                    'available_phone_numbers_toll_free': {
-                        'api_url': 'https://api.twilio.com',
-                        'api_version': '2010-04-01',
-                        'path': 'Accounts/{AccountSid}/AvailablePhoneNumbers/{ParentId}/TollFree.json',
-                        'data_key': 'available_phone_numbers',
-                        'key_properties': ['iso_country', 'phone_number'],
-                        'replication_method': 'FULL_TABLE',  # ALL for parent Address
-                        'params': {},
-                        'pagingation': 'root',
-                        'activate_version': True
-                    }
-                }
-            },
-            # pylint: disable=line-too-long
-            # Reference: https://www.twilio.com/docs/phone-numbers/api/incomingphonenumber-resource#read-multiple-incomingphonenumber-resources
-            'incoming_phone_numbers': {
-                'api_url': 'https://api.twilio.com',
-                'api_version': '2010-04-01',
-                'path': 'Accounts/{ParentId}/IncomingPhoneNumbers.json',
-                'data_key': 'incoming_phone_numbers',
-                'key_properties': ['sid'],
-                'replication_method': 'INCREMENTAL',  # Fetch ALL, filter results
                 'replication_keys': ['date_updated'],
                 'params': {},
-                'pagingation': 'none'
+                'pagination': 'root',
             },
-            # Reference: https://www.twilio.com/docs/usage/api/keys#read-a-key-resource
-            'keys': {
-                'api_url': 'https://api.twilio.com',
-                'api_version': '2010-04-01',
-                'path': 'Accounts/{ParentId}/Keys.json',
-                'data_key': 'keys',
+            'tasks': {
+                'api_url': 'https://taskrouter.twilio.com',
+                'api_version': 'v1',
+                'path': 'Workspaces/{ParentId}/Tasks',
+                'data_key': 'tasks',
                 'key_properties': ['sid'],
-                'replication_method': 'INCREMENTAL',  # Fetch ALL, filter results
+                'replication_method': 'FULL_TABLE',
```

Comment:
Same as above. `FULL_TABLE` is not compatible with the `'replication_keys': ['date_updated'],`, and it should be `INCREMENTAL` if the API endpoint is supporting it.